lib/unhangout-sockets.js

```diff
@@ -167,6 +167,17 @@ _.extend(UnhangoutSocketManager.prototype, events.EventEmitter.prototype, {
             }
         }, this));
 
+        mgr.on("store-contact", _.bind(function(socket, args) {
+            var event = this.getEvent(args.roomId);
+
+            var user = socket.user; 
+            user.set("preferredContact", args.preferredContact);
```

Comment:
Server-side validation would be nice here.  If we move the regexes into a separate library that both client/server can read, it should be easy to do.